pounders/py/tests/Test_emittance_opt.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from ibcdfo.pounders import pounders
+from ibcdfo.pounders import general_h_funs
+
+
+def call_beamline_simulation(x):
+    # In here, put your call to your simulation that takes in the
+    # parameters x and returns the three values used in the calculation of
+    # emittance.
+    # out = put_your_sim_call_here(x)
+
+    out = x.squeeze()[:3]  # This is not doing any beamline simulation!
+
+    assert len(out) == 3, "Incorrect output dimension"
+    return np.squeeze(out)
+
+
+def doit():
+    # Adjust these:
+    n = 4  # Nubmer of parameters to be optimized
+    X0 = np.random.uniform(0, 1, (1, n))  # starting parameters for the optimizer
+    nfmax = int(100)  # Max number of evaluations to be used by optimizer
+    Low = -1 * np.ones((1, n))  # 1-by-n Vector of lower bounds
+    Upp = np.ones((1, n))  # 1-by-n Vector of upper bounds
+    printf = True
+
+    # Don't adjust these:
+    hfun = general_h_funs.emittance_h
+    combinemodels = general_h_funs.emittance_combine
+    m = 3
```

Comment:
Can we get end of line comments here on what m/gtol/delta/nfs are?

Reply:
Great idea. I've added these comments now.